bin/jurassic-tube-setup.sh

```diff
@@ -34,15 +30,17 @@ echo
 read -p "Go to https://jurassic.tube/ in a browser, add a subdomain using the desired name for your subdomain, and click 'Add Subdomain'. The subdomain name is what you will use to access WC Payments in a browser. When this is done, type the subdomain name here and press enter. Please just type in the subdomain, not the full URL: " subdomain
 echo 
 
-# npm run wp option update home https://${subdomain}.jurassic.tube/
-# npm run wp option update siteurl https://${subdomain}.jurassic.tube/
-
 read -p "Please enter your Automattic/WordPress.com username: " username
 echo 
 
-${PWD}/docker/bin/jt/config.sh username ${username}
-${PWD}/docker/bin/jt/config.sh subdomain ${subdomain}
+if [ ! -f "${JT_DIR}/config.env" ]; then
+    touch "${JT_DIR}/config.env"
+fi
+
+echo "username=${username}" >> ${JT_DIR}/config.env
+echo "subdomain=${subdomain}" >> ${JT_DIR}/config.env
+echo "port=8082" >> ${JT_DIR}/config.env
```

Comment:
Hardcoding it for now. Another option would be to get it from `docker-compose.yml` but the script was not readable enough so I left it like that for now.